requirements.txt

```diff
@@ -2,3 +2,4 @@
 GitPython
 pathspec
 PyGithub
+responses
```

Comment:
Shouldn't this be only on test-requirements?

Reply:
If I installed this module and i do not have responses i got an error message saying no module name responses, thats why I have included on requirements.txt

Reply:
But that's because you have add it in the manifest, so:

- Remove it from manifest
- Remove it here
- Add it in test-requirements.txt

Reply:
done, thanks